main.go

```diff
@@ -148,6 +148,8 @@ func newApp() *cli.App {
 		systemCommand,
 		// Internal
 		internalCommand,
+		// login
+		NewLoginCommand(),
```

Comment:
coding style seems inconsistent with other commands

Reply:
@AkihiroSuda  I have to set a function here. The problem is with setting a function or a coding formatting ?

Reply:
Why does it need to be a function?

Reply:
@AkihiroSuda  fixed